lib/modules/redditUserInfo.js

```diff
@@ -0,0 +1,18 @@
+/* @flow */
+
+import { Module } from '../core/module';
+
+export const module: Module<*> = new Module('redditUserInfo');
+
+module.moduleName = 'redditUserInfoName';
+module.category = 'usersCategory';
+module.description = 'redditUserInfoDesc';
```

Comment:
Since the module only does one thing, you can remove the option and instead add `module.bodyClass = true` here.

Reply:
@andytuba Do we have any other plans for this module?

Reply:
I was thinking this module could also include options for adding RES's various userinfo buttons to the new hovercard. We could leave it as-is or migrate to the option later, but I'm inclined to prepare for more options later.

Reply:
Anyway, requiring the module for extending Reddit's hovercard to be enabled in order to disable it is a little awkward. I think this option is better suited for betteReddit.

Reply:
I agree it is a little awkward, but this module is only going to be used as a bridge before we move over to the native hovercards. So it would probably be here until the next major release, by then we will have moved across.

Reply:
Agree with the awkwardness, but also agree that we should be able to remove (or possibly repurpose) this module entirely fairly soon...for a certain definition of soon...so seems fine for now.